numpy/linalg/linalg.py

```diff
@@ -2431,6 +2432,10 @@ def norm(x, ord=None, axis=None, keepdims=False):
     The Frobenius norm is given by [1]_:
 
         :math:`||A||_F = [\\sum_{i,j} abs(a_{i,j})^2]^{1/2}`
+        
```

Comment:
Personally, I don't know whether this extra couple sentences is necessary, but that's an entirely subjective take. If anything, I would try to shorten the additional info while being clear and explicit, maybe something like:

```
Both the Frobenius and nuclear norm orders are only defined for matrices
and raise a ValueError when `x.ndim != 2`.
```

Reply:
IMO it is necessary as some people think the Frobenius norm works for vectors according to [Wolfram](https://mathworld.wolfram.com/FrobeniusNorm.html) so this extra clarification makes this more explicitly clear. I changed it to your shorter description to make it more succinct.

Reply:
Makes sense to me. One last nitpick - could it be moved so that the line about the nuclear norm is above the new sentence? I think it flows better if it's:
 - explain Frobenius norm
 - explain nuclear norm
 - explain special cases about both fro/nuc for vectors